iodata/iodata.py

```diff
@@ -301,9 +318,11 @@ def natom(self) -> int:
     @property
     def nelec(self) -> float:
         """Return the number of electrons."""
-        if self.mo is None:
-            return self._nelec
-        return self.mo.nelec
+        # When the molecular orbitals are present, they determine the number
```

Comment:
If a IOData object is created with only `charge `and `atnums `present
```python
mol = IOData(charge=-1, atnums=[8, 1, 1])
print(mol.nelec)
```
printing out `nelec` results in `None` while, after accessing `atcorenums` first, it prints out the correct value of `11.0`.
```python
mol = IOData(charge=-1, atnums=[8, 1, 1])
print(mol.atnumcores)
print(mol.nelec)
```
Is this the correct behaviour, or should `nelec` also be derived from `atnums` and the `charge` only?

Reply:
Good point, and there was a way to fix it. pfew.